docs/source/channel_update_tutorial.rst

```diff
@@ -830,10 +794,20 @@ to a protobuf so it can be properly signed and submitted to the orderer for the
     configtxlator proto_encode --input anchor_update_in_envelope.json --type common.Envelope --output anchor_update_in_envelope.pb
 
 Now that the update has been properly formatted it is time to sign off and submit it. Since this
-is only an update to Org3 we only need to have Org3 sign off on the update. As we are
-in the Org3 CLI container there is no need to switch the CLI containers identity, as it is
-already using the Org3 identity. Therefore we can just use the ``peer channel update`` command
-as it will also sign off on the update as the Org3 admin before submitting it to the orderer.
+is only an update to Org3 we only need to have Org3 sign off on the update. Run the following
+commands to make sure that we are operating as the Org3 admin:
+
+.. code:: bash
+
+  # you can issue all of these commands at once
+
+  export CORE_PEER_LOCALMSPID="Org3MSP"
+  export CORE_PEER_TLS_ROOTCERT_FILE=/opt/gopath/src/github.com/hyperledger/fabric/peer/organizations/peerOrganizations/org3.example.com/peers/peer0.org3.example.com/tls/ca.crt
+  export CORE_PEER_MSPCONFIGPATH=/opt/gopath/src/github.com/hyperledger/fabric/peer/organizations/peerOrganizations/org3.example.com/users/Admin@org3.example.com/msp
+  export CORE_PEER_ADDRESS=peer0.org3.example.com:11051
+
+We can now just use the ``peer channel update`` command to sign the update as the
+Org3 admin before submitting it to the orderer.
 
 .. code:: bash
```

Comment:
Here is the output of the peer channel update -f anchode_update... command:
2020-01-15 20:02:39.382 UTC [channelCmd] InitCmdFactory -> INFO 001 Endorser and orderer connections initialized
2020-01-15 20:02:39.420 UTC [channelCmd] update -> INFO 002 Successfully submitted channel update

Reply:
will leave to deploying CC doc